src/config.ts

```diff
@@ -180,3 +219,34 @@ export const loadConfiguration = (filepath: string = defaultFileLocation()) => {
     throw err;
   }
 };
+
+/**
+ * Writes the global config object to default location
+ */
+export const saveConfig = async (configData: string) => {
+  try {
+    const defaultDir = os.homedir() + "/.spk";
+    if (!fs.existsSync(defaultDir)) {
+      fs.mkdirSync(defaultDir);
+    }
+    fs.writeFileSync(defaultFileLocation(), configData);
+  } catch (err) {
+    logger.error(
+      `Error occurred while saving config to default location. \n ${err}`
+    );
+  }
+};
+
+/**
+ * Writes the global config object to default location
+ */
+export const writeConfigToDefaultLocation = async (filePath: string) => {
+  try {
+    const data = yaml.safeDump(readYaml<IConfigYaml>(filePath));
+    await saveConfig(data);
+  } catch (err) {
+    logger.error(
+      `Error occurred while writing config to default location. \n ${err}`
+    );
+  }
+};
```

Comment:
Ideally, would be nice to do this in a separate PR since it's not related to onboard

Reply:
Agreed. I was trying to reuse the method to save storage and table names in the config, but we can  do this in a separate PR.

Reply:
No this is fine for now! I understand the PR a lot more after testing it 👍